src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return (days * 40);
+  } else {
+    if (days < 7) {
+      return (days * 40 - 20);
+    } else {
+      return (days * 40 - 50);
+    }
```

Comment:
```suggestion
    }
    return (days * 40 - 50);
```

No need in this 'else'. This result will be returned by default if both previous checks won't pass

Reply:
all remarks has corrected!